server/node/routes.js

```diff
@@ -195,7 +195,13 @@ router.get('/products/:id', async (req, res) => {
 // Retrieve the PaymentIntent status.
 router.get('/payment_intents/:id/status', async (req, res) => {
   const paymentIntent = await stripe.paymentIntents.retrieve(req.params.id);
-  res.json({paymentIntent: {status: paymentIntent.status}});
+  const payload = {status: paymentIntent.status};
+
+  if (paymentIntent.last_payment_error) {
+    payload.last_payment_error = paymentIntent.last_payment_error.message;
```

Comment:
This is typically not a customer-facing error message. For example, on a P24 failed authentication, it's:

> The provided `PaymentMethod` has failed authentication. You can provide `payment_method_data` or a new `PaymentMethod` to attempt to fulfill this `PaymentIntent` again.

The user would be required to write their own generic error messages, but I think showing the underlying error here is better for our audience. Thoughts?

Reply:
Agreed, LGTML 👍 
![image](https://user-images.githubusercontent.com/23213994/88169205-424bca00-cc4e-11ea-89e3-87b1dfa146a5.png)